solarforecastarbiter/metrics/preprocessing.py

```diff
@@ -87,14 +90,25 @@ def resample_and_align(fx_obs, fx_series, obs_series, tz):
     with a `interval_label` the same as the `fx`, regardless of whether the
     `interval_length` of the `fx` and `obs` are the same or different.
 
+    Raises
+    ------
+    ValueError
+        If fx_obs.reference_forecast is not None but ref_series is None
+        or vice versa
+    ValueError
+        If fx_obs.reference_forecast.interval_label or interval_length
+        does not match fx_obs.forecast.interval_label or interval_length
```

Comment:
so no adjustment like we do for obs? I'm ok with that but we should clearly document and possible prohibit picking such a ref forecast on the dashboard

Reply:
should this validation go in the datamodel instead? I guess no because the metrics processing of the other fxobs  can continue and the error will be logged in the report

Reply:
I didn't want to allow adjustment of the reference forecast for the same reasons we don't touch the regular forecasts - the reference forecast could be user supplied, and it's better to not tinker with it. I'd be ok with allowing resampling of the built in reference forecasts. 

I agree it would be good to restrict the options in the dashboard PR. @lboeman is that pretty doable?

Good point about putting the validation in the datamodel instead. I could go either way.